### DISCUSSION TITLE
Customer Milestone 1 - Should we go more or less?

### DISCUSSION THREAD

**Discussion by mehmetkuzu** [bounswe/bounswe2023group2#305] [General]
_I thought a discussion item would be appropriate to finalize the content of CM1:_

Would it be better if we economize milestone 1 a little? So, for example, we can move the items related to needs and resources beyond CM1? Or do you say we can do all these in 18 days?

(There are also endpoints for these in CM1, but there are no front-end and mobile items. Will they be added or is this intentional?)

**Comment by bgmrsln** [comment]
We have 8 weeks left other than first milestone and final milestone can have some unexpected tasks. I think it would be beneficial to start for needs and resources. Since we are 5 and needs are similar to resources, it should be okay for us.

**Reply by mehmetkuzu** [reply]
Convinced. Closing the discussion.
